src/java.base/share/classes/java/nio/charset/spi/CharsetProvider.java

```diff
@@ -34,16 +34,29 @@
  *
  * <p> A charset provider is a concrete subclass of this class that has a
  * zero-argument constructor and some number of associated charset
- * implementation classes.  Charset providers may be installed in an instance
- * of the Java platform as extensions.  Providers may also be made available by
- * adding them to the application class path or by some other
- * platform-specific means.  Charset providers are looked up via the current
+ * implementation classes.  Charset providers are made available by
+ * adding them to the application module path, application class path, or by some
+ * other platform-specific means.  Charset providers are looked up via the current
  * thread's {@link java.lang.Thread#getContextClassLoader() context class
- * loader}.
+ * loader}. See {@link java.util.ServiceLoader##developing-service-providers
+ * Deploying Service Providers} for further detail on deploying a charset
+ * provider as a module or on the class path.
```

Comment:
I think it would better to say "deployed" rather than "available" here, that will align with the later text.

Reply:
I presume you meant the "available" on line 37 should be "deployed" and changed as such.